packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -38,6 +39,31 @@ contract L1StandardBridge_Initialize_Test is Bridge_Initializer {
     }
 }
 
+contract L1StandardBridge_Pause_Test is Bridge_Initializer {
```

Comment:
We need a test or tests to show that none of the withdrawal methods are callable when the bridge is paused.

Reply:
Good callout